samples/TcpClient_NarodMon/app/application.cpp

```diff
@@ -51,6 +51,7 @@ bool nmOnReceive(TcpClient& client, char *buf, int size)
 	// debug msg
 	debugf("nmOnReceive");
 	debugf("%s", buf);
+	return true;
```

Comment:
Ok, I will revert it. Agreed it isn't part of this "logical" PR, but it's here because this TCP example have led me to the error in my new library callback in the begining.